recipes/grass/build.sh

```diff
@@ -0,0 +1,57 @@
+#! /bin/bash
+
+if [ $(uname) == Darwin ]; then
+  export GRASS_PYTHON=$(which pythonw)
```

Comment:
pythonw?

Reply:
`conda-forge` does not have a `pythonw` on macOS and Windows. This may lead to a erroneous build linking to the CI default `python`. Please use `$PYTHON` to ensure the right one will be used. (I believe the build should work without the `w`.)

Reply:
`pythonw` seems to come with the *python.app* package on *conda-forge* and seems to be available for Mac. Still, I can try with just regular python through `$PYTHON`. I think *pythonw* may be necessary but I'm not sure.